apps/condo/domains/property/access/CheckPropertyWithAddressExistService.js

```diff
@@ -0,0 +1,14 @@
+/**
+ * Generated by `createservice property.CheckPropertyWithAddressExistService --type queries`
+ */
+async function canCheckPropertyWithAddressExist () {
+    return true
```

Comment:
It's not good to give access to an anonymous user 
as he can find out all addresses registered in our organization

Reply:
you're right, but as @pahaz said, it's not a big deal right now...So I left access:true for now

Reply:
Property canRead assessors should return 'false' to unpermitted user, so, ye, it's really no a big deal right now.